### PR TITLE
Add SASL description and config options

### PR DESCRIPTION
**Pull request by lwandzura**
Closes https://github.com/influxdata/docs-v2/issues/3870

Added SASL configuration parameters, descriptions and example config to Kafka event handler doc.  Update is part of the Kapacitor v1.6.4 release.

### CODE REVIEW

**Review by docmerlin — APPROVED** [review]

File: content/kapacitor/v1.6/event_handlers/kafka.md [hunk 1]
```diff
@@ -221,3 +271,29 @@ Add the handler:
 ```bash
 kapacitor define-topic-handler kafka_cpu_handler.yaml
 ```
+
+### Using SASL with Kapacitor
+
+Kapacitor can be configured to use SASL if you want to use methods of authentication other than SSL.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/event_handlers/kafka.md:277]:
```suggestion
To use an authentication method other than SSL, configure Kapacitor to use SASL. 
```

File: content/kapacitor/v1.6/event_handlers/kafka.md [hunk 2]
```diff
@@ -221,3 +271,29 @@ Add the handler:
 ```bash
 kapacitor define-topic-handler kafka_cpu_handler.yaml
 ```
+
+### Using SASL with Kapacitor
+
+Kapacitor can be configured to use SASL if you want to use methods of authentication other than SSL. 
+An example would be using Kapacitor to authenticate directly against Kafka with a username/password. 
+Multiple configuration options are available, but the most common usage is username and password as shown in the following example:
```
Comment (kelseiv) [inline content/kapacitor/v1.6/event_handlers/kafka.md:279]: Do we want to call out the other available config options?

File: content/kapacitor/v1.6/event_handlers/kafka.md [hunk 3]
```diff
@@ -221,3 +271,29 @@ Add the handler:
 ```bash
 kapacitor define-topic-handler kafka_cpu_handler.yaml
 ```
+
+### Using SASL with Kapacitor
+
+Kapacitor can be configured to use SASL if you want to use methods of authentication other than SSL. 
+An example would be using Kapacitor to authenticate directly against Kafka with a username/password.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/event_handlers/kafka.md:278]: For example, to use Kapacitor to authenticate with a username/password directly against Kafka, configure SASL.

**Review by kelseiv — APPROVED** [review]

File: content/kapacitor/v1.6/event_handlers/kafka.md (hunk 2 above)
Comment (lwandzura) [reply content/kapacitor/v1.6/event_handlers/kafka.md:279]: by far the most common is username/password, Emrys suggested just using that one

**Pull request merged**